tests/integration/ceph_upgrade_test.go

```diff
@@ -110,9 +110,15 @@ func (s *UpgradeSuite) TestUpgradeRookToMaster() {
 		cleanupFilesystemConsumer(s.helper, s.k8sh, s.Suite, s.namespace, filePodName)
 		cleanupFilesystem(s.helper, s.k8sh, s.Suite, s.namespace, filesystemName)
 		_ = s.helper.ObjectUserClient.Delete(s.namespace, objectUserID)
+<<<<<<< HEAD
 		_ = s.helper.BucketClient.DeleteObc(obcName, bucketStorageClassName, bucketPrefix, maxObject, false)
 		_ = s.helper.BucketClient.DeleteBucketStorageClass(s.namespace, objectStoreName, bucketStorageClassName, "Delete", region)
 		objectStoreCleanUp(s.Suite, s.helper, s.k8sh, s.settings.Namespace, objectStoreName)
+=======
+		_ = s.helper.BucketClient.DeleteObc(obcName, installer.ObjectStoreSCName, bucketPrefix, maxObject, false)
+		_ = s.helper.BucketClient.DeleteBucketStorageClass(s.namespace, installer.ObjectStoreName, installer.ObjectStoreSCName, "Delete")
+		objectStoreCleanUp(s.Suite, s.helper, s.k8sh, s.settings.Namespace, installer.ObjectStoreName)
+>>>>>>> fc2b8012c (object: use us-east-1 for aws go lang sdk)
```

Comment:
For these conflicts, likely we need to keep the old version but remove `, region` (line 114|115)